crates/ide_completion/src/tests/record.rs

```diff
@@ -140,25 +155,9 @@ fn main() {
 }
 "#,
         expect![[r#"
-            kw unsafe
-            kw match
-            kw while
-            kw while let
-            kw loop
-            kw if
-            kw if let
-            kw for
-            kw true
-            kw false
-            kw return
-            kw self
-            kw super
-            kw crate
-            lc foo       Foo
-            lc thing     i32
-            st Foo
-            fn main()    fn()
-            bt u32
+            fd ..Default::default()
+            fd foo1                 u32
+            fd foo2                 u32
```

Comment:
hmm that's kind of a regression, isn't it? `..` can be followed by any expression.

Reply:
That is true, this is a regression. I think the FIXME got the better of me here.